featurebyte/service/feature_list.py

```diff
@@ -273,14 +272,6 @@ async def create_document(self, data: FeatureListServiceCreate) -> FeatureListMo
             }
         )
 
-        if FeastIntegrationSettings().FEATUREBYTE_FEAST_INTEGRATION_ENABLED:
-            feature_store = await self.feature_store_service.get_document(
-                document_id=feature_data["feature_store_id"]
-            )
-            document.initialize_store_info(
-                features=feature_data["features"], feature_store=feature_store
-            )
```

Comment:
Move this to deployment flow so that this attribute is generated only the feature list is deployed.